### PR TITLE
Superseded deprecated function

### PR DESCRIPTION
**Pull request by athenamelia**
PR for superseded functions

### CODE REVIEW

**Comment by athenamelia** [comment]
- Amelia removed superseded functions and replaced them with new functions. In particular: 
- derive_var_extreme_dt() replaced by derive_vars_extreme_event()
- get_summary_records() replaced by derive_summary_records()
- derive_param_response() replaced by derive_extreme_event()

Closes #34

**Review by ddsjoberg — CHANGES_REQUESTED** [review]
@athenamelia thanks for the PR! I have two questions/comments
1. Since you added your middle initial in the DESCRIPTION file, you'll need to redocument the package and push the updated files. `devtools::document()`
2. Can you explain why this line was omitted and not replaced with anything?
image

**Comment by athenamelia** [comment]
@ddsjoberg thanks for the feedbacks! I ran devtools::document() and updated the csv file with the row for "LSTA: Last Disease Assessment by Investigator" variable.

File: tests/testthat/test-discovery.R [hunk 1]
```diff
@@ -40,7 +40,7 @@ test_that("discovery DATASET is consistent", {
   )
 })
 
-test_that("Warn for Deprecated/Supersded Functions", {
+test_that("Warn for Deprecated/Superseded Functions", {
```
Comment (ddsjoberg) [inline tests/testthat/test-discovery.R:43]: Thanks for the correction!

**Review by ddsjoberg — APPROVED** [review]
LGTM!

**Pull request merged**